templates/bundle_version.sh

```diff
@@ -0,0 +1,16 @@
+GIT=$(sh /etc/profile; which git)
+GIT_RELEASE_VERSION=$(${GIT} describe --tags --always --abbrev=0)
+COMMITS=$(${GIT} rev-list HEAD | wc -l)
+COMMITS=$(($COMMITS))
```

Comment:
Agreed, this doesn't seem to be doing anything. (POSIX) shell doesn't have types anyway. Everything is passed around as a string, though usage (e.g. `test $x -eq y`) may attempt to treat things as numbers.